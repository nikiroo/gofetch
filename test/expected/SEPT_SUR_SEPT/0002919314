    "LE STEWARD BELGE N%27A PAS ÉTÉ LICENCIÉ POUR AVOIR DONNÉ UNE  
                         INTERVIEW À LA TV"                        

  [Belgique] 26-09-2018 17:04

  o Reference: 0002919314
  o News link: http://s.7s7.be/3479986
  o Source link: 


    Par: rédaction
    
    26/09/18  -  17h04  Source:  Belga      Michael O'Leary, CEO de
    Ryanair. © belga.
    
    Le  steward  belge dont le licenciement est contesté par la CNE
    devant  le  tribunal du travail de Bruxelles n'a pas été écarté
    pour  avoir  donné  une interview à la télévision, mais pour ne
    pas  s'être  présenté  à  son  poste  à  plusieurs  reprises, a
    indiqué  mercredi  Michael  O'Leary  lors  d'une  conférence de
    presse.
    
    "Si  on  avait  voulu  le licencier lors de l'interview à la TV
    (en  juillet),  on l'aurait fait immédiatement et pas plusieurs
    mois  plus  tard",  explique le patron de Ryanair. Le steward a
    été licencié le 20 septembre.
    
    Selon  Michael  O'Leary,  si  le steward, en période d'essai, a
    été  licencié  c'est  parce  qu'il  aurait été absent plusieurs
    fois  sans  raison  durant  les  premiers mois de son embauche.
    "Quinze  jours  au  total", détaille Michael O'Leary. Il a reçu
    un  avertissement  mais a de nouveau été absent sept jours sans
    raison.  Par  ailleurs,  il  aurait échoué à neuf reprises à un
    test  de  sécurité.  "Cela  doit  être  un  record!  ",  dit M.
    O'Leary.    "Si   le   steward   s'était   repris   après   son
    avertissement, il serait toujours en poste chez nous."
    
    La  CNE  a  décidé  d'attaquer  Ryanair  devant  le tribunal du
    travail  de  Bruxelles pour le licenciement d'un steward belge,
    par   ailleurs  militant  syndical.  Selon  l'organisation,  la
    compagnie  aérienne  porte  atteinte aux libertés syndicales en
    reprochant  à son ancien employé d'avoir parlé à la presse lors
    de la grève du 25 juillet.



