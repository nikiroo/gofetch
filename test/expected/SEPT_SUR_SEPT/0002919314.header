0"Le steward belge n'a pas été licencié pour avoir donné une interview à la TV"	null/SEPT_SUR_SEPT/0002919314		70
i  [Belgique] 26-09-2018 17:04
i
i    Le  steward  belge dont le licenciement est contesté par la CNE
i    devant  le  tribunal du travail de Bruxelles n'a pas été écarté
i    pour avoir donné ...
i
