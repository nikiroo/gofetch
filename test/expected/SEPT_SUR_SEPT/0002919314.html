<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h2>"Le steward belge n'a pas été licencié pour avoir donné une interview à la TV"</h2>
	<div class='details'>([Belgique] 26-09-2018 17:04)</div>
	<br/>
    <ul>
    <li>Reference: <a href=''>0002919314</a></li>
        <li>News link: <a href='http://s.7s7.be/3479986'>http://s.7s7.be/3479986</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		Par: rédaction<br/><br/>26/09/18 - 17h04 Source: Belga    Michael O&#x27;Leary, CEO de Ryanair. © belga.<br/><br/>Le steward belge dont le licenciement est contesté par la CNE devant le tribunal du travail de Bruxelles n&#x27;a pas été écarté pour avoir donné une interview à la télévision, mais pour ne pas s&#x27;être présenté à son poste à plusieurs reprises, a indiqué mercredi Michael O&#x27;Leary lors d&#x27;une conférence de presse.<br/><br/>&quot;Si on avait voulu le licencier lors de l&#x27;interview à la TV (en juillet), on l&#x27;aurait fait immédiatement et pas plusieurs mois plus tard&quot;, explique le patron de Ryanair. Le steward a été licencié le 20 septembre.<br/><br/>Selon Michael O&#x27;Leary, si le steward, en période d&#x27;essai, a été licencié c&#x27;est parce qu&#x27;il aurait été absent plusieurs fois sans raison durant les premiers mois de son embauche. &quot;Quinze jours au total&quot;, détaille Michael O&#x27;Leary. Il a reçu un avertissement mais a de nouveau été absent sept jours sans raison. Par ailleurs, il aurait échoué à neuf reprises à un test de sécurité. &quot;Cela doit être un record! &quot;, dit M. O&#x27;Leary. &quot;Si le steward s&#x27;était repris après son avertissement, il serait toujours en poste chez nous.&quot;<br/><br/>La CNE a décidé d&#x27;attaquer Ryanair devant le tribunal du travail de Bruxelles pour le licenciement d&#x27;un steward belge, par ailleurs militant syndical. Selon l&#x27;organisation, la compagnie aérienne porte atteinte aux libertés syndicales en reprochant à son ancien employé d&#x27;avoir parlé à la presse lors de la grève du 25 juillet.
	</div>
<hr/>
</div>
</body>
